app/src/main/java/ru/beryukhov/coffeegram/MainActivity.kt

```diff
@@ -106,26 +111,33 @@ fun PagesContent(
     val currentNavigationState = navigationState
     val snackbarHostState = remember { SnackbarHostState() }
 
+    // You can scroll all the way up to the year 3000 with page count set to 36 000 --> (3000 * 12)
```

Comment:
Not important: It seems, that we could optimize it by limiting the start year as lets say 1970 or 1000 for the simplicity.